### PR TITLE
fix(docs): update github.md app permissions

### PR DESCRIPTION
**Pull request by leonschwanitz**
Fixes: https://github.com/nextauthjs/next-auth/issues/7777

> _NOTE_:
>
> - It's a good idea to open an issue first to discuss potential changes.
> - Please make sure that you are _NOT_ opening a PR to fix a potential security vulnerability. Instead, please follow the [Security guidelines](https://github.com/nextauthjs/.github/blob/main/SECURITY.md) to disclose the issue to us confidentially.

## 🧢 Checklist

- [x] Documentation
- [x] Tests
- [x] Ready to be merged

## 🎫 Affected issues

Please [scout and link issues](https://github.com/nextauthjs/next-auth/issues) that might be solved by this PR.

Fixes: INSERT_ISSUE_LINK_HERE

## 📌 Resources

- [Security guidelines](https://github.com/nextauthjs/.github/blob/main/SECURITY.md)
- [Contributing guidelines](https://github.com/nextauthjs/.github/blob/main/CONTRIBUTING.md)
- [Code of conduct](https://github.com/nextauthjs/.github/blob/main/CODE_OF_CONDUCT.md)
- [Contributing to Open Source](https://kcd.im/pull-request)

### CODE REVIEW

**Comment by ndom91** [comment]
@leonschwanitz if you still want this merged, can you please add this to the `main` branch? The provider docs will be in the `packages/core/providers/github.ts` file in typedoc :+1: 

Thanks!

**Comment by socketopp** [comment]
My adapter Drizzle requires email on the users. 
When I signin with certain github accounts, it fails with AdapterError due to Drizzle throwing `NOT NULL constraint failed: user.email`.

If adding that permission, would that fix the issue I stumbled across?

**Comment by ndom91** [comment]
> My adapter Drizzle requires email on the users. 
> When I signin with certain github accounts, it fails with AdapterError due to Drizzle throwing `NOT NULL constraint failed: user.email`.
> 
> If adding that permission, would that fix the issue I stumbled across? 
> 

Can't say for sure, but it does sound plausible. Also maybe you need to add an additional scope to your github oauth app

**Comment by socketopp** [comment]
> > My adapter Drizzle requires email on the users.
> > When I signin with certain github accounts, it fails with AdapterError due to Drizzle throwing `NOT NULL constraint failed: user.email`.
> > If adding that permission, would that fix the issue I stumbled across?
> 
> Can't say for sure, but it does sound plausible. Also maybe you need to add an additional scope to your github oauth app

I added the aforementioned email scope and now it works. I am suprised that no one else have had this problem. Another solution would be to make the users email nullable but this solution is better for sure.

**Comment by ndom91** [comment]
@leonschwanitz can you please change this PR to point at the `main` branch? If there are too many changes, it might be easier to just recreate the PR then :/

File: docs/docs/providers/github.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ https://developer.github.com/apps/building-oauth-apps/authorizing-oauth-apps
 
 https://github.com/settings/apps
 
+:::warning
```
Comment (balazsorban44) [inline docs/docs/providers/github.md:18]:
```suggestion
:::info
```

**Pull request merged**